k8s.gcr.io/images/k8s-staging-build-image/images.yaml

```diff
@@ -511,23 +526,36 @@
     "sha256:eca6e44f2f2a9c2d5079ac81732e15a91d8d9bbd4aae9c1246c7f9e235a30886": ["v1.14.2-4"]
     "sha256:ef3d86412d9335e7d587aebef9bc004bd42845516fee11e830a49d5e55aaaad0": ["v1.15.10-legacy-1"]
     "sha256:f393f7b488a9488fc1dff42ba24e2674f9a2d962c7dedd93a457aaf414ab956e": ["v1.15.15-legacy-1"]
+    "sha256:f429e4596e165fa541022955efcfa4f1416aad87c57ffa29129d767d70cc91e5": ["v1.21.0-go1.16.14-buster.0"]
     "sha256:f9f569e6e1da0fa6ac441b625e599bead8ab10f480ad2a84966cd4a0091e76a8": ["v1.12.17-2"]
 - name: kube-cross-amd64
   dmap:
+    "sha256:49e5820282dd7986d822332599db22efa2fae1164ca6f4d60c00c55f03a3d70c": ["v1.21.0-go1.16.14-buster.0"]
+    "sha256:4ddcb8ede9428cd0d5c39850a556d64afc99aae011fe7df378b5307b90022d83": ["v1.22.0-go1.16.14-buster.0"]
     "sha256:552ad5b1f93115c0d2ef8b1792d4790865245f24493a258093656cb61dfd4250": ["v1.21.0-go1.16.13-buster.0"]
     "sha256:562b6a0c860210449402e814fbeba4c9c70d4d294e1368fcfa23e2ed31740ea8": ["v1.23.0-go1.17.6-bullseye.0"]
+    "sha256:63a4efa7e0364e488681f008a139975bb6271d30e68a3fbda0217c170fee4798": ["v1.23.0-go1.17.7-bullseye.0"]
     "sha256:8349bdeec9c4ff9614d0fd555ec5c287854b0f1ee28910ac60592a3b2b65e1de": ["v1.22.0-go1.16.13-buster.0"]
     "sha256:947bb9b6626136e642f6cd808be2e3d52955740fc536dd8dd68e386463c02c68": ["v1.24.0-go1.17.6-bullseye.0"]
+    "sha256:e658e6e7cd36de83ff00d73d28ccaf8a9b22d2a0f67f7216e0f7b3a2a1c88ba5": ["v1.24.0-go1.17.7-bullseye.0"]
 - name: kube-cross-arm64
   dmap:
     "sha256:04be8a7a2816ad71921f8260042c8e241f1a01620db6dba8a1200d5bec1ed7a9": ["v1.22.0-go1.16.13-buster.0"]
     "sha256:2690b37115520a85eb37307b5713535b805a0a16be7fb205822a73d8026de15f": ["v1.23.0-go1.17.6-bullseye.0"]
+    "sha256:4f44932b5a054f88535261d90a6dd5fa49490deb63560b2ec37f6ba3dbe8c1ea": ["v1.24.0-go1.17.7-bullseye.0"]
     "sha256:5fa3d546a8cff366d54e2dac983eaeae15e83bc3820689256943effb0569f601": ["v1.24.0-go1.17.6-bullseye.0"]
+    "sha256:7b2f4b7b32c57b4f2af661050347a7b67a4f6241abcbc906dc1132ab50b78df9": ["v1.22.0-go1.16.14-buster.0"]
+    "sha256:8e31bdcd438181fdb35019ac63dd598e5817152991bedbc92dd866ae3937f4dc": ["v1.23.0-go1.17.7-bullseye.0"]
+    "sha256:b2b661e1bc11e6af6c22c650b4f62045ab272ecb0e89aa6b0c5fcb100ec076aa": ["v1.21.0-go1.16.14-buster.0"]
```

Comment:
One thing that I realized while promoting images -- do we actually need v1.21.0 images? Since 1.21 is in maintenance mode, do we want to update Go?

cc @kubernetes/release-engineering

Reply:
Uuuh 1.21 is not yet in maintenance mode. It enters EOL in june :)